srcpkgs/mold/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'mold'
+pkgname=mold
+version=1.0.2
+revision=1
+archs="i686 x86_64"
+build_style=gnu-makefile
+hostmakedepends="cmake pkg-config"
+makedepends="libstdc++-devel openssl-devel zlib-devel"
+short_desc="A Modern Linker."
```

Comment:
short_desc shouldn't start with an article (a, an, the) and shouldn't end with a period

Reply:
I'll keep it in mind